Geocoding

herrerop opened the pull request:
Added Geocoding functionality

Compiled address and coordinates for every unit available

herrerop committed:
Geocoding

Added Geocoding functionality

Compiled address and coordinates for every unit available

arredond reviewed (changes requested):
Before merging, remove the output from the Jupyter Notebook and add the driver logs and pycache files to the gitignore so they won't be tracked

herrerop committed:
Revert "Geocoding"

This reverts commit 254ba205548bd4099fd0c842b70046e2a5181a18.

herrerop committed:
geocodign

added gitignore arguments

removed outputs from jupyter notebook

updated utils file with new functions and dependencies
